NEWS.md

```diff
@@ -1,5 +1,37 @@
 # OSBuild Composer - Operating System Image Composition Services
 
+## CHANGES WITH 28:
+
+  * OSBuild Composer can now build CentOS Stream 8 images!
+
+  * OSBuild Composer can now upload images to VMWare!
+
+  * Cloudapi contains new /openapi.json and /version endpoints. The already
+    existing `POST /compose` endpoint now supports specification of additional
+    packages.
+
+  * Blueprints now support definition of alternative kernels, such as kernel-rt,
+    using the `customizations.kernel.name` key.
+
+  * Attempt to use the customization.kernel key in a blueprint for the OSTree
+    image type now results in error, because the customization was ignored.
```

Comment:
```suggestion
  * Attempt to use the `customization.kernel.append` key in a blueprint for the OSTree
    image type now results in error, because the customization was ignored.
```

I realise this was unclear in my original description of the change, but it's actually the `append` key that's ignored.

Reply:
Thanks, I'll fix it.